R/render.R

```diff
@@ -54,6 +54,10 @@ render <- function(input,
                        encoding = encoding)
       outputs <- c(outputs, output)
     }
+    if (length(output_file) > 1) {
```

Comment:
Good point. I did think about it, and `file.rename()` will signal an error if `length(output_file) != length(outputs)`, so I decided not to add the check here.